src/lib/api/Livechat.ts

```diff
@@ -22,12 +22,12 @@ import ApiBase from './api'
 export default class ApiLivechat extends ApiBase {
   credentials: ILivechatRoomCredentialAPI = {} as any
   login (guest: INewLivechatGuestAPI | any) { return this.grantVisitor(guest) }
-  async config (params?: ILivechatTokenAPI) { return (await this.get('livechat/config', params, false)).config }
+  async config () { return (await this.get('livechat/config', { token: this.credentials.token }, false)).config }
```

Comment:
@MarcosSpessatto, we can't change the signature of this method because it's the only one where the `token` parameter is optional.
In all other methods is OK.